solidity/dashboard/src/contracts.js

```diff
@@ -235,3 +240,17 @@ export function getBondedECDSAKeepFactoryAddress() {
 export function getTBTCSystemAddress() {
   return getContractAddress(TBTCSystem)
 }
+
+const getOldTokenStakingArtifact = async () => {
+  if (
+    process.env.NODE_ENV === "development" ||
+    getFirstNetworkIdFromArtifact() === "3"
+  ) {
+    // For local and ropsten network we want the mock old `TokenStaking` contract.
+    return (
+      await import("@keep-network/keep-core/artifacts/OldTokenStaking.json")
+    ).default
+  }
+
+  return (await import("./old-contracts-artifacts/TokenStaking.json")).default
```

Comment:
What does this `default` do? I'm just curious.

Reply:
`await import(*)` returns `module object` that contains all exports. pls [see](https://javascript.info/modules-dynamic-imports#the-import-expression).